compact_str/src/repr/mod.rs

```diff
@@ -501,6 +537,12 @@ impl<'a> MutStrongRepr<'a> {
     }
 }
 
+#[derive(Debug)]
+enum StrongIntoRepr {
```

Comment:
nit: maybe `StrongOwnedRepr`? makes it a little more general in case we use this for more than just `into_string()` in the future